packages/mint-components/src/components/sqm-share-link/ShareLink.feature

```diff
@@ -54,3 +54,30 @@ Feature: Share Link
     Then an "USER_REFERRAL_PROGRAM_ENGAGEMENT_EVENT" analytic event is fired
     And it is for "program-a"
     And it has share medium "DIRECT"
+
+
+  @ui
+  Scenario Outline: user can edit the alignment of the share link text
+    Given a user is editing the share link component
+    Then they see "Align text" props
+    And the default value is "left"
+    When they change the option to <option>
+    Then they see the text in <position>
+    Examples:
+      | option | position |
+      | left   | left     |
+      | center | center   |
+      | right  | right    |
+
+  @ui
+  Scenario Outline: The position of the copy button can be changed
```

Comment:
Is the default backwards compatible @noahwc @lillyszatmary ?

Reply:
yes, new props are optional and the default value is the same as the old hardcoded one